documentation/docs/src/clients/socks5-client.md

```diff
@@ -134,22 +140,53 @@ The `config.toml` file contains client configuration options, while the two `pem
 The generated files contain the client name, public/private keypairs, and gateway address. The name `<client_id>` in the example above is just a local identifier so that you can name your clients.
 
 #### Configuring your client for Docker
+
 By default, the native client listens to host `127.0.0.1`. However this can be an issue if you wish to run a client in a Dockerized environment, where it can be convenenient to listen on a different host such as `0.0.0.0`.
 
 You can set this via the `--host` flag during either the `init` or `run` commands.
 
 Alternatively, a custom host can be set in the `config.toml` file under the `socket` section. If you do this, remember to restart your client process.
 
-
-
 ### Running the socks5 client
 
-You can run the initalised client by doing this:
+You can run the initialised client by doing this:
 
 ```
 ./nym-socks5-client run --id docs-example
 ```
 
+## Automating your socks5 client with systemd
```

Comment:
nice! has this been tested?

Reply:
yes, and running fine!

Reply:
I'm worried about starting to add stuff like `journalctl` commands because i dont want the docs to get bloated - if anything I want them to get a bit more streamlined.. do you think its necessary?